content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+nearby clusters until you address the issue.
+
+You can configure inter-cluster communication based on the
+[network](#network-models) and the options supported by your cloud provider. For
+example, if two clusters reside on the same underlying network, you can enable
+cross-cluster communication by simply configuring firewall rules.
+
+## Network models
+
+Many production systems require multiple networks or subnets for isolation
+and high availability. Istio supports spanning a service mesh over a variety of
+network topologies.
+
+### Single network
+
+In the simplest case, a service mesh operates over a single fully
+connected network. A single network model satisfies the following conditions:
+
+- The network consists of a number of connected subnets without any
+  overlap in IP or VIP ranges.
+- All {{< gloss >}}workload instance{{< /gloss >}}s can reach each other directly
+  without an Istio gateway.
+
+A single network allows Istio to configure service consumers in a uniform
+way across the mesh with the ability to directly address workload instances.
+
+{{< image width="75%"
+    link="./single-net.svg"
+    alt="A service mesh with a single network"
+    title="Single network"
+    caption="A service mesh with a single network"
+    >}}
+
+### Multiple networks
+
+You can span a single service mesh across multiple networks; such a
+configuration is known as **multi-network**.
+
+Multiple networks afford the following capabilities beyond that of single networks:
+
+- An existing network topology
+- Overlapping IP or VIP ranges for **service endpoints**
+- Crossing of administrative boundaries
+- Fault tolerance
+- Scaling of network addresses
+
+In this model, the workload instances in different networks can only reach each
+other through one or more [Istio gateways](/docs/concepts/traffic-management/#gateways).
+Istio uses **partitioned service discovery** to provide consumers a different
+view of {{< gloss >}}service endpoint{{< /gloss >}}s. The view depends on the
+network of the consumers.
+
+{{< image width="75%"
+    link="./multi-net.svg"
+    alt="A service mesh with multiple networks"
+    title="Multi-network deployment"
+    caption="A service mesh with multiple networks"
+    >}}
+
+## Mesh models
+
+Istio supports two mesh models:
+
+- Single mesh
+- Multiple meshes, also known as **multi-mesh**
+
+### Single mesh
+
+The simplest Istio deployment is a single mesh. Within a mesh, service names are
+unique. For example, only one service can have the name `foo/mysvc`.
+Additionally, workload instances all share a common identity and authentication
+infrastructure and can authenticate each other. With a single mesh you can do
+the following tasks:
+
+- Span [one or more clusters](#cluster-models)
+- Span [one or more networks](#network-models)
+- Use [namespaces for tenancy](#namespace-tenancy)
+
+### Multiple Meshes
+
+You can deploy multiple meshes; such a configuration is known as
+**multi-mesh**.
```

Comment:
So this is what I was proposing to call a mesh federation in https://github.com/istio/istio.io/pull/4788.
Notice that I had been defining it as a noun, "a mesh federation" similar to a "federation of states"? I see that below federation is used as a verb.

So, in retrospect I think I like this better. Using the name "multi-mesh" for the noun lets us then define "mesh federation" to be the act of federating two or more meshes in a multi-mesh deployment. I think that would be less ambiguous.

We need to add glossary entries for "Multi-mesh Deployment" and "Mesh Federation" to make sure it's totally clear how they relate.

Reply:
@frankbu yes, that's exactly my thinking as well!

Reply:
I'll introduce new glossary entries for both `multi-mesh` used as a noun, and `mesh federation` as the following verb:
```
Mesh federation is the act of federating two or more meshes in a multi-mesh deployment.
```
@smawson @nmittler @frankbu Would that definition work?

Reply:
I think we should leave "federating" out of the definition.  Maybe:

```
Mesh federation is the act of exposing services between meshes in a multi-mesh
deployment. In a federation, each mesh may expose a subset of their services
for one or more other meshes to consume. This affords service communication
across mesh boundaries.
```

Reply:
That's a circular definition of federate :)

I think something along the lines of what @smawson said [here](https://github.com/istio/istio.io/pull/4813#discussion_r316395847) would work.
```
Mesh federation is the act of connecting two meshes together by sharing trust and identity information as well as service discovery information.
```

Reply:
@frankbu 

> That's a circular definition of federate :)

Which definition are you referring to? [Rigs's](https://github.com/istio/istio.io/pull/4813#discussion_r316810773) or [mine](https://github.com/istio/istio.io/pull/4813#discussion_r316815058)?

> I think something along the lines of what @smawson said here would work.

The problem with that definition in my mind is that it doesn't really paint the picture of the objective of mesh federation. The point is that they can share services. The phrases `connecting two meshes together` and `sharing trust and identity... as well as service discovery information` doesn't really make it explicit. In addition, saying that they share `service discovery information` sounds like they expose everything, which very likely won't be the case.

Reply:
@nmittler, I meant Rigs' definition was circular `federation is the act of federating` :)

I see what you mean about @smawson's definition. Your suggestions works for me, although maybe we should avoid using the word federation as both a verb and a noun.

Should we also mention that the meshes being federated are isolated from each other and have different service definitions and independent administrative control, or maybe link `multi-mesh deployment` to the definition where that's all covered?
```
Mesh federation is the act of exposing services between meshes in
a [multi-mesh deployment](link). Each mesh may expose 
a subset of its services for one or more other meshes to consume.
This affords service communication across mesh boundaries.
```

Reply:
@frankbu 

> Should we also mention that the meshes being federated are isolated from each other and have different service definitions and independent administrative control, or maybe link multi-mesh deployment to the definition where that's all covered?

I don't think so. That's effectively the definition of a service mesh, which is defined elsewhere on istio.io. We should be able to assume that as read for the purposes of this document.

Also, I think you're rewording of my definition looks good!